api_volontaria/apps/position/models.py

```diff
@@ -0,0 +1,184 @@
+from datetime import datetime, timedelta
+import pytz
+from babel.dates import format_date
+from django.conf import settings
+from django.db import models
+from django.db.models.signals import post_save, pre_delete
+from django.dispatch import receiver
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+from django.contrib.auth import get_user_model
+from dry_rest_permissions.generics import authenticated_users
+# from api_volontaria.email import EmailAPI
+
+User = get_user_model()
+
+
+class Position(models.Model):
+    """
+    This class represents a position that an organization can advertise and that volunteer can apply to.
+    """
+
+    class Meta:
+        verbose_name = _("Position")
+        verbose_name_plural = _("Positions")
+
+    name = models.CharField(
+        verbose_name=_("Name"),
+        max_length=100,
+    )
+
+    description = models.TextField(
+        verbose_name="Description",
+    )
+
+    hourly_wage = models.FloatField(
+        verbose_name=_("Hourly wage"),
+    )
+    
+    weekly_hours = models.FloatField(
+        verbose_name=_("Weekly hours"),
+    )
+    
+    minimum_duration_commitment = models.FloatField(
+        verbose_name=_("Minimum duration commitment"),
+    )
```

Comment:
The `minimum_duration_commitment` should be rename to clearly define the unity of measure (ex: `minimum_days_commitment`.

Frontend application will be able to use different structure of form, calculate the duration in days and return it to the API, but we need to keep it clear in the API that we wait a value in days only.

Reply:
done